apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/setup-xm-cloud.md

```diff
@@ -0,0 +1,183 @@
+![Create Project and Environment Step 3 - Setup Respository](https://sitecorecontenthub.stylelabs.cloud/api/public/content/3adbe7059311434d8b7900d691990e22?v=08e2526a) 
+5. Provide a name for the repository e.g.xm-cloud-tutorial-series and click the Continue button. 
+6. Enter a name for the environment e.g. DEV  
+![Create Project and Environment Step 4 - Provide Environment details](https://sitecorecontenthub.stylelabs.cloud/api/public/content/7f69f87c697148afabc6e55526588d4f?v=751f0044) 
+7. Select whether this is a production environment or not. You select “No” 
+8. Select whether you want to autodeploy on push to the repository. Select “Yes”. This enables the CI/CD pipeline from your main branch. This can be adjusted later. 
+9. Press the Continue button. 
+10. On the 5th step review your selections and press the “Start deployment” button
+![Create Project and Environment Step 5 - Review your selections](https://sitecorecontenthub.stylelabs.cloud/api/public/content/d5cda1bf224c4f99b508fe612e527590?v=6c0f0076) 
+11. The deployment starts, and provisioning and build run in parallel.  
+![XM Cloud Deploy Logs - Deployment is running](https://sitecorecontenthub.stylelabs.cloud/api/public/content/81ff5cf671f24d948aeb0dfb6d769411?v=4abce4be) 
+**A note on provisioning and build:**  
+*The Provisioning sets up all the resources you need to run the XM Cloud instance, while the Build is building the software solution you have cloned to your personal repository. This will take a few minutes. Read on for more information on what is going on 'under the hood' as your project sets up.*  
+
+
+
+## What is provisioned for XM Cloud 
+
+So, how does this work? Referencing the figure below, let’s walk through the different bits and pieces of XM Cloud.  
+![XM Cloud Architecture](https://sitecorecontenthub.stylelabs.cloud/api/public/content/21cbe922e28b4896b1692be7c85ae6c4?v=c1ed41cb) 
+
+ 
+To begin, XM Cloud contains a Content Management instance (CM / purple box) that publishes content to a geographically distributed delivery layer called *Edge* (yellow box). Edge offers a GraphQL endpoint to then query for the content coming from XM Cloud. Your app is typically connected with, and can run with whatever host you are rendering your front end on, such as Vercel, Netlify, AWS, Azure, or another vendor.  
+
+
+In order to utilize the WYSIWYG editing experience, an internal rendering host, or *editing host* (green box), is delivered with XM Cloud. This editing host is managed by Sitecore, and runs a Node server that can host any Node-based application.  
+
+The main way of accessing XM Cloud is through the *Sitecore Cloud Portal* (solid outlined box, 'Sitecore Cloud').  This GUI allows you to manage and access your different Sitecore Apps and Users. For an in-depth exploration of the Sitecore Cloud Portal, navigate [here](https://doc.sitecore.com/portal/en/developers/sitecore-cloud-portal/introduction-to-the-sitecore-cloud-portal.html).  
```

Comment:
Can we use a CTA box for the link? I think we have some controls available to us on the Dev Portal.

Reply:
Decision: We don't want them leaving, so keep as is.